pages/mainboard.js

```diff
@@ -1,11 +1,14 @@
 import Board from "@/components/Board";
+import RandomMoveEngine from "@/components/RandomMoveEngine";
 import styled from "styled-components";
```

Comment:
It is a convention to first import external components (Link, styled-components, etc.) and then the own... Just for readability....

Reply:
Thank you, I´ll fix that.